vnext/docs/ParityStatus.md

```diff
@@ -0,0 +1,38 @@
+# Component & Module API Parity Status
+This doc reports the status progress of core components modules/APIs integration for vnext. The components and modules tracked here will be in alignment with the [React Native Lean Core](https://github.com/facebook/react-native/issues/23313) effort.
+
+Ones in **bold** are in the milestone we're currently targetting.
+
+We are in **Milestone 2.**
+
+### Components
+Below are a list of components and their status towards parity between the lean core components you'd find on iOS and Android and the functionality we have today.
+
+|Component| Status | Issues | Milestone 2| Milestone 3 |
```

Comment:
I don't think Milestone 2 and 3 columns are needed. The issues will be tracked against a milestone which should indicate when the API is lighting up.  And you have bolded the issues that are being tracked in the current milestone, so that's good enough information, I think.

Reply:
I removed them and added a c# status column instead.